github/client.go

```diff
@@ -970,6 +985,29 @@ func (client *Client) apiClient() *simpleClient {
 	return &simpleClient{
 		httpClient: httpClient,
 		rootUrl:    apiRoot,
+		extraHeader: func(r *http.Header) {
+			if headers, err := git.ConfigAll(fmt.Sprintf("http.%s://%s.extraheader", client.Host.Protocol, client.Host.Host)); err == nil {
+				url, _ := url.Parse(fmt.Sprintf("%s://%s", client.Host.Protocol, client.Host.Host))
+				cookies := []*http.Cookie{}
+
+				for _, header := range headers {
+					keyValue := strings.Split(header, ":")
+					key, value := strings.TrimSpace(keyValue[0]), strings.TrimSpace(keyValue[1])
+
+					if strings.ToUpper(key) == "COOKIE" && strings.Contains(value, "=") {
```

Comment:
Tip: you can use `strings.EqualFold()` to compare strings case-insensitive

Reply:
Thanks, never saw it